ktor-shared/ktor-serialization/ktor-serialization-jackson/jvm/src/JacksonConverter.kt

```diff
@@ -80,8 +102,27 @@ public class JacksonConverter(private val objectmapper: ObjectMapper = jacksonOb
  *
  * You can learn more from [Content negotiation and serialization](https://ktor.io/docs/serialization.html).
  */
+@Deprecated("This will be removed.", level = DeprecationLevel.HIDDEN)
+public fun Configuration.jackson(
+    contentType: ContentType = ContentType.Application.Json,
+    block: ObjectMapper.() -> Unit = {}
+) {
+   jackson(contentType, true, block)
+}
+
+/**
+ * Registers the `application/json` content type to the [ContentNegotiation] plugin using Jackson.
+ *
+ * You can learn more from [Content negotiation and serialization](https://ktor.io/docs/serialization.html).
+ *
+ * @param contentType the content type to send with request
+ * @param streamRequestBody if set to true, will stream request body, without keeping it whole in memory.
+ * This will set `Transfer-Encoding: chunked` header.
+ * @param block a configuration block for [ObjectMapper]
+ */
 public fun Configuration.jackson(
     contentType: ContentType = ContentType.Application.Json,
+    streamRequestBody: Boolean = true,
```

Comment:
Should we think about adding a configuration class instead? so In the future, we can add more properties without deprecations